torch/_dynamo/convert_frame.py

```diff
@@ -52,6 +52,7 @@
 log = logging.getLogger(__name__)
 guards_log = torch._logging.getArtifactLogger(__name__, "guards")
 bytecode_log = torch._logging.getArtifactLogger(__name__, "bytecode")
+recompilations_log = torch._logging.getArtifactLogger(__name__, "recompilations")
```

Comment:
bikeshed: recompiles (is shorter)

Reply:
I'll go one better "recompile"

Reply:
oh you meant the alias, yeah I'll change both to recompiles